test/remote/db_tests.js

```diff
@@ -156,27 +169,25 @@ describe('remote db', function() {
           return db.updateSessionToken(sessionToken, 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:41.0) Gecko/20100101 Firefox/41.0')
         })
         .then(function() {
-          return db.sessionToken(tokenId)
+          return db.sessions(account.uid)
         })
-        .then(function(sessionToken) {
-          assert.equal(sessionToken.lastAccessTime, sessionToken.createdAt, 'session token was not updated')
-          return sessionToken
+        .then(function(sessionTokens) {
+          assert.equal(redisSetSpy.lastCall, null, 'session token was not updated')
+          return db.sessionToken(tokenId)
         })
         .then(function(sessionToken) {
           sessionToken.lastAccessTime -= TOKEN_FRESHNESS_THRESHOLD
           return db.updateSessionToken(sessionToken, 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:41.0) Gecko/20100101 Firefox/41.0')
         })
-        .then(function() {
+        .then(function(sessionTokens) {
+          assert.ok(redisSetSpy.lastCall, 'session token was updated')
```

Comment:
We should assert that the arguments to this call were what we expect them to be.